deployments/admin-api/monitoring.yaml

```
@@ -0,0 +1,128 @@
+          },
+          {
+            "title": "Audit Log Writes",
+            "targets": [
+              {
+                "expr": "rate(audit_log_writes_total{job='admin-api'}[5m])"
+              }
+            ]
+          }
+        ]
+      }
+    }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)